build-system/tasks/helpers.js

```diff
@@ -484,8 +484,8 @@ async function esbuildCompile(srcDir, srcFilename, destDir, options) {
     let map = result.outputFiles.find(({path}) => path.endsWith('.map')).text;
 
     if (options.minify) {
-      ({code, map} = await minify(code, map));
-      map = await massageSourcemaps(map, options);
+      ({code, map: minifiedMap} = await minify(code));
```

Comment:
nit: why the rename?

Reply:
Because we have 2 map variables, and one is already named `map`.